lib/configs/api-gateway/schema.ts

```diff
@@ -488,48 +488,33 @@ export const articleListSchema: JsonSchema = {
                             "author": {
                                 type: JsonSchemaType.STRING,
                             },
-                            "summary": {
```

Comment:
We need this to display a brief content of an article in the list.

Reply:
Maybe set it for empty string now？